mythril_multiboot2/src/services.rs

```diff
@@ -37,4 +37,29 @@ impl VmServices for Multiboot2Services {
     fn acpi_addr(&self) -> Result<HostPhysAddr> {
         Ok(HostPhysAddr::new(0))
     }
+
+    fn rsdp(&self) -> Result<crate::acpi::rsdp::RSDP> {
+        // Get a v1 tag if possible
+        match self.info.rsdp_v1_tag() {
+            Some(tag) => {
+                let oem_id = tag.oem_id().map(|found| found.as_bytes());
+                match oem_id {
+                    Some(id) => {
+                        let mut arr: [u8; 6] = [0; 6];
+                        arr.copy_from_slice(&id[0..6]);
+                        return Ok(crate::acpi::rsdp::RSDP::V1 {
+                            oemid: arr,
+                            rsdt_addr: tag.rsdt_address() as u32,
+                        });
+                    },
+                    None => {}
+                }
+            },
+            None => {}
+        }
+
+        // TODO: Get a v2 tag if possible
```

Comment:
nit: could you change this to `TODO(ntegan)`, so people know who to ping if they hit this section?